exercicios/para-casa/.gitignore

```
@@ -0,0 +1 @@
+*node_modules/
```

podemos adicionar o .env  aqui também, o que acha?
```suggestion
*node_modules/
*.env
```

---

Feito, Mayyyy. beijos